### DISCUSSION TITLE
I need request.post example

### DISCUSSION THREAD

**Discussion by pystar1007** [FlareSolverr/FlareSolverr#791] [General]
Hello Gould you share example of how to use request.post?

--
Must be a string with application/x-www-form-urlencoded. Eg: a=b&c=d
--

What does this mean?

**Comment by ngosang** [comment]
POST Example:
```
curl -L -X POST 'http://localhost:8191/v1' \
-H 'Content-Type: application/json' \
--data-raw '{
            "cmd": "request.post",
            "url": "https://httpbin.org/post",
            "postData": "param1=value1&param2=value2"
        }'
```

The value of `postData` must be encoded as `application/x-www-form-urlencoded`. You can not post a JSON for example.

**Reply by 33357** [reply]
API requires json data, how to convert it
